src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.netcore.cs

```diff
@@ -12013,13 +12013,86 @@ public static Vector512<T> Invoke(Vector512<T> x, Vector512<T> y)
             public static Vector512<T> Invoke(Vector512<T> x, Vector512<T> y, Vector512<T> z) => (x * y) + z;
         }
 
+        /// <summary>(x * y) + z</summary>
+        internal readonly struct MultiplyAddEstimateOperator<T> : ITernaryOperator<T> where T : IFloatingPointIeee754<T>
+        {
+            public static T Invoke(T x, T y, T z) => MultiplyAddEstimate(x, y, z);
+            public static Vector128<T> Invoke(Vector128<T> x, Vector128<T> y, Vector128<T> z) => MultiplyAddEstimate(x, y, z);
+            public static Vector256<T> Invoke(Vector256<T> x, Vector256<T> y, Vector256<T> z) => MultiplyAddEstimate(x, y, z);
+            public static Vector512<T> Invoke(Vector512<T> x, Vector512<T> y, Vector512<T> z) => MultiplyAddEstimate(x, y, z);
```

Comment:
Is it worth having the above broken into separate operators like it is? Can we not just have it here and have callers that need it use `MultiplyAddEstimateOperator<T>.Invoke` to avoid the additional calls/inlining?